src/python/WMCore/FwkJobReport/XMLParser.py

```diff
@@ -334,16 +339,54 @@ def perfSummaryHandler():
     while True:
         report, node = (yield)
         summary = node.attrs.get('Metric', None)
+        module = node.attrs.get('Module', None)
         if summary is None:
             continue
+        site = None
+        if module == 'XrdSiteStatistics':
+            site = summary
+            summary = 'XrdSiteStatistics'
+
         # Add performance section if it doesn't exist
         if not hasattr(report, summary):
             report.section_(summary)
         summRep = getattr(report, summary)
 
         for subnode in node.children:
-            setattr(summRep, subnode.attrs['Name'],
-                    subnode.attrs['Value'])
+#             setattr(summRep, subnode.attrs['Name'],
+#                     subnode.attrs['Value'])
+            name = subnode.attrs['Name']
+            value = subnode.attrs['Value']
+            if isinstance(value, str):
```

Comment:
I would suggest to make a new function/method for this as well (castMetricValue?). Then we can easily cover it with unit test as well.

Reply:
done